src/build.sh

```diff
@@ -110,6 +109,59 @@ locate_llvm_exec()
     fi
 }
 
+get_download_url()
+{
+    manifest=$EnlistmentRoot/build/components/$1/cgmanifest.json
```

Comment:
>$EnlistmentRoot/build/components/$1/cgmanifest.json [](http://example.com/codeflow?start=13&length=51)

nit: this path and instances of `$manifest` should be enclosed in quotes incase `$EnlistmentRoot` contains spaces.